features/environment.py

```diff
@@ -18,12 +21,20 @@
     "SANDBOX": SANDBOX_INT_BASE_URL,
 }
 
+CIS2_USERS = {
+    "prescriber": {"user_id": "656005750107", "role_id": "555254242105"},
+    "dispenser": {"user_id": "555260695103", "role_id": "555265434108"},
+}
+
 REPOS = {
     "EPS-FHIR": "https://github.com/NHSDigital/electronic-prescription-service-api",
     "PFP-APIGEE": "https://github.com/NHSDigital/prescriptions-for-patients",
 }
 
-# This will need rework when the pack includes additional products to test
+CERTIFICATE = os.getenv("CERTIFICATE")
```

Comment:
These - combined with the dotenv import - suggest that a .env file is needed, including the relevant environment variables. Is that something that needs adding to the README?

Reply:
Good spot. 
The values need to remain private so I'm not sure where we could store them so future devs could utilise them.

Reply:
So a mention in the README, do you think? With some guidance on where to get those from? Probably one for Sean when he's back.

Reply:
Apologies, I forgot to ammend. Looking at it now

Reply:
I have added a section to the readme regarding running locally including the need for environment variable